containers/LocationMarker.js

```diff
@@ -21,14 +21,17 @@ class LocationMarker extends PureComponent {
   handleDragEnd(event) {
     const { onDragEnd, location } = this.props;
 
+    this.forceUpdate();
+
     onDragEnd(event, location);
   }
 
   render() {
     const { onDrag, pinColor, location } = this.props;
 
     return (
-      <MapView.Marker.Animated
+      <MapView.Marker
+        key={`${this.props.locationKey}${Date.now()}`}
```

Comment:
The outer object is pure. I wonder if that is why the rendering what not happening as expected.